### ISSUE TITLE
Splitting documents based on “SEPARATOR-PAGES” 

### ISSUE THREAD

**Issue by Toskache** [jbarlow83/OCRmyPDF#510]
**Describe the issue**
To scan and OCR a bunch of different documents it would be great if OCRmyPDF would be able to split documents using “seperation-pages” (with special barcode or special text…) which could inserted into the stack before scanning. That would bring workflows and productivity to a next level.

**Example file**
You can find two sample-files for orientation-failproof-two-side-page separator pages here: [https://cloud.hermann.ac/s/9PKsMaST95i6yJH](url)

**Expected behavior**
If the content of a page is just [Content of the separator-page], than discard that page, save all previous pages and start a new document.

**Comment by jbarlow83** [comment]
I agree this would be "nice to have". But this strikes me as a custom requirement specific to your workflow rather than a feature of general interest. (Other people might use barcodes, and have different rules for naming output files.)

If you are interesting in contracting with me to develop this feature we could discuss that by email.

**Comment by Toskache** [comment]
Ok, I understand. How can I contact you by mail?

**Comment by jbarlow83** [comment]
jim@purplerock.ca

**Comment by jbarlow83** [comment]
Closing as out of scope - we're not implementing features that remove pages or split documents, since these can be destructive. That kind of operation should exist at a different step in your document pipeline.
